initramfs-init.in

```diff
@@ -381,6 +383,31 @@ fi
 # zpool reports /dev/zfs missing if it can't read /etc/mtab
 ln -s /proc/mounts /etc/mtab
 
+if echo $@ | grep -q "dasd" || echo $@ | grep -q "qeth" ; then
+	for mod in dasd_mod dasd_eckd_mod dasd_fba_mod qeth qeth_l2 qeth_l3; do
+		modprobe $mod
+	done
+	_tmpfile="$(mktemp)"
+	echo $@ | tr A-Z a-z | xargs -n 1 | grep '=' > "$_tmpfile"
+	. "$_tmpfile" > /dev/null 2>&1
```

Comment:
Is it really needed to pollute global namespace with arbitrary variables…?!

Reply:
I updated the code to use current getopts/myopts code instead of this.

Reply:
Thanks.